lib/action_network_rest/people.rb

```diff
@@ -19,5 +19,37 @@ def unsubscribe(id)
       response = client.put_request "#{base_path}#{url_escape(id)}", request_body
       object_from_response(response)
     end
+
+    def find_id_by_email(email)
+      url_encoded_filter_string = url_escape("email_address eq '#{email}'")
+      response = client.get_request "#{base_path}?filter=#{url_encoded_filter_string}"
+      person_action_network_id_from_query_response(response)
```

Comment:
Would it be worth having a method in this gem that returns the whole person block (i.e. `response.body[:_embedded]['osdi:people'].first`) and not just the ID? (and then maybe we could stick the `action_network_id` in it like we do in other places)

Reply:
sure, that makes sense, because if we later need more info we don't have to break the existing method to add some fields